Project.toml

```diff
@@ -13,6 +13,7 @@ julia = "1"
 Dates = "1"
 OrderedCollections = "1.4"
 StructTypes = "1.8"
+JSONSchema = "< 1.4.1"
```

Comment:
Test failure noted above is coming from 1.4.1 of JSONSchema, https://github.com/fredo-dedup/JSONSchema.jl/compare/v1.4.0...v1.4.1, something in that diff is the likely cause.

Reply:
weird. Created issue https://github.com/matthijscox/JSONSchemaGenerator.jl/issues/5 so we can move on for now.